src/main/scala/mesosphere/marathon/core/deployment/impl/DeploymentActor.scala

```diff
@@ -246,9 +245,9 @@ object DeploymentActor {
     Props(new DeploymentActor(
       deploymentManagerActor,
       killService,
```

Comment:
Same goes for the `killService` here - I think _Scheduler_ can replace it. Ideally *all* of the deployment actor only use the new _Scheduler_

Reply:
The will with [phase 3](https://jira.mesosphere.com/browse/MARATHON-8292). This diff could also move the kill service. Should I move it in this diff as well?